Chapter_3/3_04_EnergyLossGain.tex

```diff
@@ -138,7 +138,7 @@ \section{Energy loss and gain}\label{sec:3.4}
 
 It is quite common for the rf voltage of a storage ring to have a sinusoidal variation with time. In such cases we would have that
 \begin{align}\label{eq:3.36}
-	V(\tau) = \widehat{V} \sin(\omega_{rf}(\tau+\tau_0))
+	V(\tau) = \widehat{V} \sin(\omega_{rf}\tau+\phi_0)
 \end{align}
 where $\widehat{V}$ is called the ``peak rf voltage'' and $\omega_{rf}\tau_0$  is called the ``synchronous rf phase angle''. With our assumptions
```

Comment:
missing substitution here too ?

Reply:
I've changed all \tau_0 to \phi_0 in chapter 3.

Reply:
@Gabrielrezende-asc , @otavio-silveira , can we merge this PR ?